django/thunderstore/community/models/community.py

```diff
@@ -66,12 +66,24 @@ class Community(TimestampMixin, models.Model):
     @property
     def total_package_count(self) -> int:
         # TODO: Implement as a cached value with background updates
-        return -1
+        listings = self.package_listings.active()
+
+        if self.require_package_listing_approval:
+            listings = listings.approved()
+
+        return listings.count()
 
     @property
     def total_download_count(self) -> int:
         # TODO: Implement as a cached value with background updates
-        return -1
+        # TODO: also figure out more efficient way to handle this
+
+        listings = self.package_listings.active()
+
+        if self.require_package_listing_approval:
+            listings = listings.approved()
+
+        return sum([l.total_downloads for l in listings])
```

Comment:
Can we add a check here which returns `-1` on non-debug environments until the cache-based solution exists? Otherwise it'll probably end up blocking deployment as it's a potential DOS vector I wouldn't feel particularly good about deploying.